_proposals/pup-0001.rst

```diff
@@ -0,0 +1,328 @@
+including links to responses via the archive. Collaboration is also easy with a collaborator sending
+revisions to the author as a patch or a new pull request against the proposal.
+
+
+Voting
+^^^^^^
+
+The +1, +0, -0, -1 feedback process is used, with votes sent as responses via the
+announcement/discussion mailing list. The voting has the following meanings:
+
+* +1: "Will benefit the project and should definitely be adopted."
+* +0: "Might benefit the project and is acceptable."
+* -0: "Might not be the right choice but is acceptable."
+* -1: "Not the right choice and should definitely not be adopted." This type of feedback must come
+  with a statement of objective concerns explaining what harm will come from adopting the proposal.
+
+Note that votes can come from anyone, not just core devs [#f1]_. This is directly modeled after [the
+Django process](https://docs.djangoproject.com/en/dev/internals/contributing/bugs-and-features/#how-we-make-decisions)
+who modeled their process after the Python and Apache communities.
+
+
+Deciding
+^^^^^^^^
+
+* If an obvious consensus arises via voting, the proposal is ratified. This is a bit vague, but it
+  works for the Django community.
+* Discussion on the thread can confirm if consensus is obvious or not.
+* Votes come from anyone, not just core devs [#f1]_.
+* A -1 from a core dev [#f1]_ is considered a veto vote and will block ratification until the vote
+  is recast.
+* Core devs are expected to consider and echo serious feedback from the community, especially any -1
+  votes from community members.
+* Consensus is not always possible. If obvious consensus is not reached, then the core devs [#f1]_
+  decide.
+
+
+Proposal Requirements
+=====================
+
+This is a modified version of [the Foreman template](https://github.com/theforeman/rfcs/blob/master/0000-template.md).
+
+::
+
+    ---
+    title: <title goes here>
+    author: <author or authors named>
+    ---
+
+    Summary
+    =======
+
+    One paragraph explanation of the feature.
+
+
+    Motivation
+    ==========
+
+    Why are we doing this? What problem is it designed to solve? What is the expected outcome?
+
+
+    Detailed Design
+    ===============
+
+    This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with the
+    project to understand, and for somebody familiar with the codebase to implement. This should get
+    into specifics and corner cases, and include examples of how the feature is used.
+
+
+    Drawbacks
+    =========
+
+    Why should we not do this?
+
+
+    Alternatives
+    ============
+
+    What other designs have been considered? What is the impact of not doing this?
+
+    Be creative here. Brainstorm with others. No idea is too outlandish to mention. Include ideas
+    that are "obviously" bad; even one sentence about why something was ruled out is valuable.
+
+
+    Unresolved Questions
+    ====================
+
+    What parts of the design are still TBD?
```

Comment:
Is it worth spelling out "to be determined" to the benefit of folks that might not be familiar with this abbreviation?